drainer/syncer.go

```diff
@@ -394,6 +399,10 @@ ForLoop:
 				beginTime := time.Now()
 				s.itemsWg.Add(1)
 				lastAddComitTS = binlog.GetCommitTs()
+
+				log.Info("add ddl item to syncer, you can add this commit ts to `ignore-txn-commit-ts` for skip this ddl if need",
```

Comment:
Maybe only output this line when `Sync` failed in `if err != nil`?

Reply:
it's ok to log info for every ddl binlog statement(this is important and will not too much).
but it better only log the part `you can add this commit ts to ignore-txn-commit-ts..` only when failing to exec the ddl, but the `Sync` is async, it returns success only means add to syncer success `but not` successfully synced to downstream, so log it for every ddl now.